src/RuleImportEntity.php

```diff
@@ -192,13 +192,15 @@ public function getAdditionalCriteriaDisplayPattern($ID, $condition, $pattern)
 
         $crit = $this->getCriteria($ID);
         if (count($crit) && $crit['field'] == '_source') {
-            if ($pattern != 'GLPI') {
+            if ($pattern != 'GLPI' && $pattern != 'NATIVE_INVENTORY') {
+                $name = $pattern;
```

Comment:
I misread the diff. To not break previous behaviour, code should be:
```suggestion
            if ($pattern == 'GLPI') {
                $name = $pattern;
```